components/style/stylist.rs

```diff
@@ -1635,6 +1635,8 @@ impl CascadeData {
     }
 }
 
+/// Iterator over `PerOriginCascadeData`, from highest level (user) to lowest
+/// (user agent).
```

Comment:
Add a comment to note that we rely on that to get animations, and potentially other stuff?